core/benchmarks/PropertiesCloneBenchmark-jdk11-results.txt

```diff
@@ -6,35 +6,35 @@ OpenJDK 64-Bit Server VM 11.0.4+11-LTS on Linux 3.10.0-862.3.2.el7.x86_64
 Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
 Empty Properties:                         Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-SerializationUtils.clone                              0              0           0          0.1       13755.0       1.0X
-Utils.cloneProperties                                 0              0           0          3.5         285.0      48.3X
+SerializationUtils.clone                              0              0           0          0.1       11539.0       1.0X
+Utils.cloneProperties                                 0              0           0          1.7         572.0      20.2X
```

Comment:
Seems only this is slower?

Reply:
Thank you for review. Do you mean JDK11 is slower? Here, JDK8 took more time. The following is the JDK8 result in this PR. JDK11 took `572.0ns` per row while JDK8 did `608.0ns` per row.
```
SerializationUtils.clone                              0              0           0          0.1       13640.0       1.0X
Utils.cloneProperties                                 0              0           0          1.6         608.0      22.4X
```

Reply:
oh, I meant previously 48.3X -> now 20.2X, so seems the speed up of Utils.cloneProperties seems less.

Reply:
Yes. The ratio is different due to the underlying machine change.